service/controller/clusterapi/v29/resource/tccp/testdata/case-0-basic-test.golden

```diff
@@ -0,0 +1,941 @@
+
```

Comment:
I generally like the idea of this golden file here - I am just concerned that we will have some trouble while developing as different changes on different resources could cause it to be outdated.

Reply:
If changes in code elsewhere in `aws-operator` changes the rendered CF template, the test will break. If this was intentional then one should just update this template. If it was accidental, the code should be fixed.

If there are changes that don't change the output of this template rendering, then I don't see any problem in that.